scripts/waitonserver.sh

```diff
@@ -1,10 +1,16 @@
 #!/usr/bin/env bash
+set -e
 
-echo 'waiting on healthcheck' >&2
+if [[ -z "${NEAR_PID}" ]]; then
+  echo "NEAR_PID is undefined"
+  exit 1
+fi
+
+echo "waiting on healthcheck NEAR_PID = ${NEAR_PID}" >&2
```

Comment:
This whole thing can be written as

    echo "waiting on health check; NEAR_PID = ${NEAR_PID:?undefined}" >&2

Reply:
@mina86 Thanks, I like it. That's a much better way to do it.